src/features/user/MFV2/ForestProgress/index.tsx

```diff
@@ -0,0 +1,140 @@
+import { useEffect, useState, useMemo } from 'react';
+import targetBarStyle from './ForestProgress.module.scss';
+import TargetsModal from './microComponents/TargetsModal';
+import { useMyForestV2 } from '../../../common/Layout/MyForestContextV2';
+import EmptyProgress from './microComponents/EmptyProgress';
+import Skeleton from 'react-loading-skeleton';
+import ForestProgressItem from './microComponents/ForestProgressItem';
+
+interface TargetProps {
+  handleOpen: () => void;
+  treesDonated: number;
+  areaRestored: number;
+  areaConserved: number;
+}
+
+const ProgressBars = ({
+  handleOpen,
+  treesDonated,
+  areaRestored,
+  areaConserved,
+}: TargetProps) => {
+  const {
+    contributionsResult,
+    treeTarget,
+    restoreTarget,
+    treeChecked,
+    restoreChecked,
+    conservTarget,
+    conservChecked,
+  } = useMyForestV2();
+
+  const conditionsToShowBar = (
+    target: number,
+    checked: boolean,
+    treesDonated: number
+  ): boolean => {
+    if (treesDonated > 0 || (target > 0 && checked)) {
+      return true;
+    } else {
+      return false;
+    }
+  };
```

Comment:
1. `shouldShowBar` may be a more appropriate name here.
2. Why is the last argument called `treesDonated`, when that isn't applicable for `areaRestored` or `areaConserved`? `score` or `contributedUnits` may be more relevant.
3. You could truncate the complete `if` statement to `return (treesDonated > 0 || (target > 0)  && checked)`?